README.md

```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+If you leave out the `_BRANCH`, the repository default branch is used - your commit has to be on the specified branch.
+
+You should always test and check changes before deploying a new commit, as many people of the community have commit rights and there is currently no test or release process defined.
```

Comment:
Shouldn't this be part of the contribution guidelines? If you meant to add this for users, maybe replace "before deploying a new commit" with "before using a new commit of the community-packages in your modules" or something like that.

Reply:
I meant this for users - adjusted the comment accordingly